<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>778</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>NFTG</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
border-radius : 10px;
border-style : outset;
border-width: 2px;
border-color : #1545C1;
height: 30px;
font-size: 25px;
font-weight: bold;
color: #1545C1;</string>
    </property>
    <property name="text">
     <string>GENERATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40 48pt &quot;Arial&quot;;
color: #000;
background-color : none</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:48pt; font-weight:40; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600;&quot;&gt;NFT ART GENERATOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCommandLinkButton::hover{
background: none;
border: 3px solid #1545C1;
border-radius: 10px;
padding-bottom: 10px;

}</string>
    </property>
    <property name="text">
     <string>Select Traits Folder</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Downloads/Group 16.png</normaloff>../Downloads/Group 16.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>670</y>
      <width>941</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
background-color : rgb(200, 200, 200);
color: rgb(170,85,127);
border-style: solid;
border-radius: 10px;
text-align: center;
font-size: 18px;
}
QProgressBar::chunk{
border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.136, x2:1, y2:0.114, stop:0 rgba(56, 66, 255, 255), stop:1 rgba(255, 215, 219, 255));
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Calibri&quot;;
background-color: none;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Please enter limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;
background-color : none</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="config_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
border-radius : 10px;
border-style : outset;
border-width: 2px;
border-color : #1545C1;
height: 30px;
font-size: 25px;
font-weight: bold;
color: #1545C1;</string>
    </property>
    <property name="text">
     <string>CONFIRM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="generate_amount">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
border-radius : 10px;
border-style : outset;
border-width: 2px;
border-color : #1545C1;
height: 30px;
font-size: 16px;
padding: 5px 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="height">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>353</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
border-radius : 10px;
border-style : outset;
border-width: 2px;
border-color : #1545C1;
height: 30px;
padding: 5px 5px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="width">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>352</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
border-radius : 10px;
border-style : outset;
border-width: 2px;
border-color : #1545C1;
height: 30px;
font-size: 16px;
padding: 5px 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>305</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size : 24px;
background-color : none;
</string>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size : 24px;
background-color : none;
</string>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-20</y>
      <width>1091</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/nftjpg.jpg</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>116</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color: #00298A;
border-radius : 10px;
opacity: 0.2;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>256</width>
      <height>380</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
background-color : none;
padding: 5px 5px;
font-size: 15px

}
QListWidget::item { 
                                    border-bottom: 1px solid #E2F3FE;
                                    border-top: 1px solid #E2F3FE;
                                    border-radius: 5px;
                                    padding: 2px 2px;
                                    margin-bottom: 5px;
									margin-right: 5px;
                                    width:100%; 
                                    height:50px;
									color: #fff;
                                    cursor:pointer; 
                                    box-shadow:0.1em 0em 0.4em 0em grey;
                                    background-color: #1545C1;
									
                                    }
QScrollBar:vertical {              
            border: none ;
            background: ;
            width:8px;
        }
QScrollBar::handle:vertical {
            background-color: #2359E3;
            max-height: 20px;
			border-radius: 10px;
        }
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
   </widget>
   <widget class="QLabel" name="layer_label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: none;
color: #fff;
font-size: 15px
</string>
    </property>
    <property name="text">
     <string> LAYER LIST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mx_img_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>max image genrate possiblity</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_counter">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>image counter</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>commandLinkButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>config_btn</zorder>
   <zorder>generate_amount</zorder>
   <zorder>height</zorder>
   <zorder>width</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>frame</zorder>
   <zorder>listWidget</zorder>
   <zorder>layer_label</zorder>
   <zorder>mx_img_value</zorder>
   <zorder>img_counter</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
